packages/story-editor/src/components/library/panes/media/local/deleteDialog.js

```diff
@@ -83,13 +83,13 @@ function DeleteDialog({ mediaId, type, onClose }) {
   const imageDialogTitle = __('Delete Image?', 'web-stories');
   const videoDialogTitle = __('Delete Video?', 'web-stories');
   const imageDialogDescription = __(
-    'You are about to permanently delete this image from your site. ' +
-      'The image will appear broken in any content that uses it. ',
+    'You are about to permanently delete this image from your site.' +
+      'The image will appear broken in any content that uses it.',
     'web-stories'
   );
   const videoDialogDescription = __(
-    'You are about to permanently delete this video from your site. ' +
-      'The video will appear broken in any content that uses it. ',
+    'You are about to permanently delete this video from your site.' +
```

Comment:
This one should not be flagged as incorrect. Only the one on the next line.

Otherwise there is no spaces between the two sentences.

```suggestion
    'You are about to permanently delete this video from your site. ' +
```

Reply:
https://github.com/GoogleForCreators/web-stories-wp/pull/11868/files#r913143597